javascript/chronometer.js

```diff
@@ -1,34 +1,53 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
     // ... your code goes here
+    if (callback) {
+      this.intervalId = setInterval(callback, 1000);
+    } else {
+      this.intervalId = setInterval(() => this.currentTime++, 1000);
+    }
```

Comment:
You are calling the callback but not incrementing the currentTime, it should be like this:
```
this.intervalId = setInterval(() => {
   if (callback) {
callback()
}
    this.currentTime++
}, 1000) 
```